ui/dashboards/src/components/PanelDrawer/PanelPreview.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Box } from '@mui/material';
+import { PanelEditorValues } from '../../context/DashboardProvider/panel-editing';
+import { Panel, PanelProps } from '../Panel';
+
+export function PanelPreview({ name, description, kind, spec, groupIndex }: PanelEditorValues) {
+  const previewValues: PanelProps = {
+    definition: {
+      kind: 'Panel',
```

Comment:
is it always going to be a `Panel` kind here? if so, do we need to remove `kind` from `PanelEditorValues`?

Reply:
Yeah that's true, could be a good way to simplify things further!